core/src/com/unciv/ui/pickerscreens/PickerScreen.kt

```diff
@@ -6,48 +6,45 @@ import com.unciv.UncivGame
 import com.unciv.ui.utils.*
 import com.unciv.ui.utils.AutoScrollPane as ScrollPane
 
-open class PickerScreen(val disableScroll: Boolean = false) : CameraStageBaseScreen() {
+open class PickerScreen : CameraStageBaseScreen() {
 
     internal var closeButton: TextButton = Constants.close.toTextButton()
     protected var descriptionLabel: Label
-    protected var rightSideGroup = VerticalGroup()
+    private var rightSideGroup = VerticalGroup()
     protected var rightSideButton: TextButton
-    private var screenSplit = 0.85f
 
     /**
      * The table displaying the choices from which to pick (usually).
      * Also the element which most of the screen realestate is devoted to displaying.
      */
     protected var topTable: Table
     var bottomTable:Table = Table()
-    internal var splitPane: SplitPane
     protected var scrollPane: ScrollPane
 
     init {
+        val bottomTableHeight = 150f
         bottomTable.add(closeButton).pad(10f)
 
         descriptionLabel = "".toLabel()
         descriptionLabel.wrap = true
-        val labelScroll = ScrollPane(descriptionLabel,skin)
+        val labelScroll = ScrollPane(descriptionLabel)
         bottomTable.add(labelScroll).pad(5f).fill().expand()
 
         rightSideButton = "".toTextButton()
         rightSideButton.disable()
         rightSideGroup.addActor(rightSideButton)
 
         bottomTable.add(rightSideGroup).pad(10f).right()
-        bottomTable.height = stage.height * (1 - screenSplit)
 
         topTable = Table()
         scrollPane = ScrollPane(topTable)
 
-        scrollPane.setScrollingDisabled(disableScroll, disableScroll)
-        scrollPane.setSize(stage.width, stage.height * screenSplit)
-
-        splitPane = SplitPane(scrollPane, bottomTable, true, skin)
-        splitPane.splitAmount = screenSplit
-        splitPane.setFillParent(true)
-        stage.addActor(splitPane)
+        val pickerScreenTable = Table()
+        pickerScreenTable.add(scrollPane).height(stage.height - bottomTableHeight - 2f).row()
+        pickerScreenTable.addSeparator()
```

Comment:
The `SplitPane` handle is a different colour than the separator - I liked the version with a little less contrast better. Alternatively, one could argue that maybe _all_ separators should use the SplitPane handle colour from the skin... Like ... like [here](https://github.com/SomeTroglodyte/Unciv/blob/b531404cb18db401cd0e1bdab7b38c110706d86a/core/src/com/unciv/ui/pickerscreens/ModManagementScreen.kt#L120).

Reply:
SplitPane is not very good if you want to assign an exact width or height for widget

Reply:
I believe you.Also using SplitPane when you know it's never gonna be dragged is some unnecessary overhead.  I was talking about visual appearance only.